src/client/index.js

```diff
@@ -3,11 +3,10 @@ import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import tableReducer from './reducers';
-import fakeData from './data/data';
```

Comment:
Should we remove `./data/data` as well?

Reply:
Yesp

Reply:
I just want to have default data that can import :P